### ISSUE TITLE
getPage fails to get all posts (skips some periods)

### ISSUE THREAD

**Issue by nlch** [pablobarbera/Rfacebook#113]
I've been using the package to download posts from public newspaper pages and I've run into a small issue (other than that it's been working wonderfully).

It seems that when I try to download posts within a certain period, there are significant gaps in the download (for example if I download from june 2016 to septmeber 2016 I get all posts on june but it skips july completely and half of august). If I check the facebook page on that missing period there are many public posts that are not being downloaded.

Small example:
#284515247529 should be Corriere della Sera
posts <- getPage(page = "284515247529", token, n = 1000,
                           since = "2016-06-01", until = "2016-07-30",
                           feed = TRUE, reactions = FALSE)

The response download posts from June but skips July.

**Comment by kevin0932** [comment]
Hi @nlch , did you figure out why the function getPage() skips some periods? I met with similar situation just starting from 5 days ago. I am really appreciated if you can share some information or solutions you have got.

when I now use the function getPage() with no time filter, I can only get several hundreds of posts (used to be thousands, much more than those I could retrieve now) and they are in two clearly separated time ranges, for example, some old posts created before 10/12/2009 and the rests are between 10/04/2017-today 01/05/2017.

Thank you so much.

**Comment by nlch** [comment]
Hi @kevin0932, I haven't been able to figure out what the issue is. At the time I tested different things and I still had missing periods (which I only noticed because I had downloaded the same data before). I can't be certain but I think this happened right after one of the Graph API updates, yet I didn't find any mention of this anywhere else.
The strange thing is, that when I tried to get the likes and comments of posts in those blank periods by their IDS I could gather that information with no problem. 

I would have to try again and see what happens now without the since-until parameters tho...
Hopefully that's not a permanent change.
